visidata/undo.py

```diff
@@ -37,8 +37,14 @@ def undo(vd, sheet):
     if not vd.options.undo:
         vd.fail("options.undo not enabled")
 
-    # don't allow undo of first command on a sheet, which is always the command that created the sheet.
-    for i, cmdlogrow in enumerate(sheet.cmdlog_sheet.rows[:0:-1]):
+    cmdlogrows = itertools.dropwhile(lambda r: r.longname == 'set-option', sheet.cmdlog_sheet.rows)
+    # skip the first remaining command, to exclude it from undo,
+    # because it is always the one that created the sheet
+    try:
```

Comment:
Is this just `list(cmdlogrows)[1:]`? If so I'd rather use that.  (If at some point on any PR you'd like me to just go ahead and make changes myself, let me know, I know I have some quirky taste sometimes.)

Thank you so much for diving into this whole issue, by the way.

Reply:
Sure, feel free to make any changes yourself. I don't mind and it's faster turnaround time.